portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -21,24 +21,34 @@
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
 import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private ArrayList<String> data;
+  private ArrayList<String> comments;
 
   @Override
   public void init() {
-    data = new ArrayList<String>();
-    data.add("item one");
-    data.add("item two");
-    data.add("item three");
+    comments = new ArrayList<String>();
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String userComment = request.getParameter("text-input");
+    if (userComment == null) {
+      userComment = "";
+    }
+    comments.add(userComment);
```

Comment:
When the userComment is null, resetting to empty string seems like it could cause unintended behaviors. 

Would it be better to simply ignore the empty user comments?

Reply:
oh that's a good idea! I agree that blank user comments should probably be ignored. Let me go change that.